juntagrico/dao/jobdao.py

```diff
@@ -76,5 +76,10 @@ def get_promoted_jobs():
                                           time__gte=timezone.now()).order_by('time')[:Config.promomted_jobs_amount()]
 
     @staticmethod
-    def upcomming_jobs_for_member(member):
-        return Job.objects.filter(time__gte=timezone.now(), assignment__member=member).distinct()
+    def upcoming_jobs_ordered_by_time_ascending_for_member(member):
```

Comment:
I would keep the name. For backwards compatibility, but also because I think "sorted by time" is the default way we want to get the members jobs and not something that needs to be highlighted.

Reply:
As you wish. I just thought while correcting the type I'll change the name to reflect what's actually been done in this PR.